src/engine/controls/cuecontrol.cpp

```diff
@@ -1800,30 +2087,128 @@ void CueControl::hotcueFocusColorNext(double v) {
     pCue->setColor(colorPalette.nextColor(*color));
 }
 
+void CueControl::setCurrentSavedLoopControl(HotcueControl* pControl) {
```

Comment:
Can you elaborate why "saved" is misleading? It is saved as a hotcue. Otherwise I think you can easily confuse if with hotcue loops (beatloops set from hotcues via `hotcue_X_gotoandloop`, like you can in the CUE LOOP mode in Serato)